### DISCUSSION TITLE
Is there a way to deploy hubs on other cloud providers?

### DISCUSSION THREAD

**Discussion by lukewhitingdev** [mozilla/hubs#3049] [Hubs Cloud Questions] [answered]
Just a quick question since I know the aws cloud deployment comes through the aws marketplace and i havent used the other ones.

**Comment by robinkwilson** [comment, marked as answer]
Great question!

Hubs Cloud can be deployed on AWS and DigitalOcean (alpha).

From @misslivirose's answer in #2825 : we currently only have routes to support deployments on AWS and Digital Ocean, which is in early access. While there is potential for the service to be deployed on other systems, we are focusing on AWS and Digital Ocean for the time being. The code required to DIY your own hosting setup is all open source, There is more information in this thread about self-hosting and/or minimizing deployment costs: #2699
